kubernetes/crd/domain-crd.yaml

```diff
@@ -26,10 +26,63 @@ spec:
           type: object
           description: DomainSpec is a description of a domain.
           properties:
+            configuration:
+              type: object
+              description: Models and overrides affecting the WebLogic domain configuration
+              properties:
+                opss:
+                  type: object
+                  description: Configuration for OPSS based security
+                  properties:
+                    keyWalletConfigMap:
+                      type: string
+                      description: The name of the config map containing the OPSS
+                        key wallet file
+                    walletSecret:
+                      type: object
+                      description: Secret containing OPSS key passphrase.
+                      properties:
+                        name:
+                          type: string
+                        namespace:
+                          type: string
+                model:
+                  type: object
+                  description: Model in image model files and properties
+                  properties:
+                    encryptionSecret:
```

Comment:
I think it's likely that the encryptionSecret setting will be very rarely used and overally not very useful for model-in-image purposes -- so maybe move it to the end so it's not the first thing people see? Similarly, maybe move it to the last element in the java schema code too. 
 Also, please add sentence 'Required when WDT model files are encrypted."   (The reason it's rare is that the encryption is there to hide things like passwords, but, in Kubernetes land, passwords will come from macros that reference secrets.)

Reply:
These files are auto-generated.  I think we have them sorted alphabetically, but I can check.  Either way, let's adjust (if we need to) in another PR.